pandas/core/series.py

```diff
@@ -3635,6 +3636,61 @@ def reorder_levels(self, order):
         result.index = result.index.reorder_levels(order)
         return result
 
+    def explode(self) -> "Series":
+        """
+        Create new Series expanding a list-like column.
+
```

Comment:
Can you add a longer description here as well? Eg based on the reshape doc you added "Transforms each element of a list-like to a row, replicating the index values:"